backend-shared/config/db/postgres-integration.go

```diff
@@ -8,14 +8,18 @@ import (
 )
 
 // function to connect to Postgres
-func connectToDatabase(verbose bool) (*pg.DB, error) {
+func connectToDatabase(verbose bool, dbName string, args ...int) (*pg.DB, error) {
```

Comment:
I think we can change this function signature to
```
func connectToDatabase(verbose bool, dbName string, port int) (*pg.DB, error) {
```
The issue is that `...int` implies that we also support multiple ports (>=2), which we do not.

Another option would be to do this with 2 functions:
```go
func connectToDatabase(verbose bool, dbName string) (*pg.DB, error) {
    return connectToDatabase(vebose, dbName, DEFAULT_PORT)
}
```
and
```go
func connectToDatabaseWithPort(verbose bool, dbName string, port int) (*pg.DB, error) {
```

Reply:
Totally makes sense, and yes we can do that as well. I'll fix this up quickly!